src/Task/ConvertTranslatableTask.php

```diff
@@ -0,0 +1,268 @@
+                if (is_a($extension, FluentExtension::class, true)) {
+                    $classes[] = $base;
+                    break;
+                }
+            }
+        }
+        return array_unique($classes);
+    }
+
+    public function run($request)
+    {
+        // Extend time limit
+        set_time_limit(100000);
+
+        // we may need some privileges for this to work
+        // without this, running under sake is a problem
+        // maybe sake could take care of it ...
+        Security::setCurrentUser(
+            DefaultAdminService::singleton()->findOrCreateDefaultAdmin()
+        );
+
+        $this->checkInstalled();
+        $this->withTransaction(function (ConvertTranslatableTask $task) {
+            Versioned::set_stage(Versioned::DRAFT);
+            $classes = $task->fluentClasses();
+            $tables = DB::get_schema()->tableList();
+            if (empty($classes)) {
+                Debug::message('No classes have Fluent enabled, so skipping.', false);
+            }
+
+            foreach ($classes as $class) {
+                /** @var DataObject $class */
+
+                // Ensure that a translationgroup table exists for this class
+                $baseTable = DataObject::getSchema()->baseDataTable($class);
+                $groupTable = strtolower($baseTable . "_translationgroups");
+                if (isset($tables[$groupTable])) {
+                    $groupTable = $tables[$groupTable];
+                } else {
+                    Debug::message("Ignoring class without _translationgroups table $class", false);
+                    continue;
+                }
+
+                // Disable filter if it has been applied to the class
+                if (singleton($class)->hasMethod('has_extension')
+                    && $class::has_extension(FluentFilteredExtension::class)
+                ) {
+                    $class::remove_extension(FluentFilteredExtension::class);
+                }
+
+                // Select all instances of this class in the base table, where the Locale field is not null.
+                // Translatable has a Locale column on the base table in SS3, but Fluent doesn't use it. Newly
+                // created records via SS4 Fluent will not set this column, but will set it in {$baseTable}_Localised
+                $instances = DataObject::get($class, sprintf(
+                    '"%s"."Locale" IS NOT NULL',
+                    $baseTable
+                ));
+
+                foreach ($instances as $instance) {
+                    /** @var DataObject $instance */
+
+                    // Get the Locale column directly from the base table, since the SS ORM will not include it
+                    $instanceLocale = SQLSelect::create()
+                        ->setFrom($baseTable)
+                        ->setSelect('Locale')
+                        ->setWhere(['ID' => $instance->ID])
```

Comment:
`'"ID"'` vs `'ID'` since you need to SQL escape this fragment. Postgres will complain. I'd even add basetable to be consistent.

```
->setWhere(["\"{$baseTable}\".\"ID\"" => $instance->ID])
```